src/js/index.js

```diff
@@ -0,0 +1,156 @@
+class Calculator {
+  constructor() {
+    this.firstDigit = "";
+    this.secondDigit = "";
+    this.operation = "";
+    this.result = 0;
+    this.init();
+  }
+
+  init() {
+    this.total = document.getElementById("total");
+    this.digits = document.querySelectorAll(".digit");
+    this.operations = document.querySelectorAll(".operation");
+    this.acBtn = document.querySelector(".modifier");
+    this.bindEvents();
+  }
+
+  convertToInteger() {
+    this.firstDigit = parseInt(this.firstDigit, 10);
+    this.secondDigit = parseInt(this.secondDigit, 10);
+  }
+
+  bindEvents() {
+    this.digits.forEach(digitButton => {
+      digitButton.addEventListener("click", e => this.handleDigit(e));
+    });
+
+    this.operations.forEach(digitButton => {
+      digitButton.addEventListener("click", e => this.handleOperation(e));
+    });
+
+    this.acBtn.addEventListener("click", () => {
+      this.reset();
+      this.renderTotal(this.result);
+    });
+  }
+
+  handleDigit(e) {
+    // 첫번째 수 입력
+    const targetDigit = e.target.textContent;
+    if (this.validateFirstDigit(targetDigit)) {
+      this.updateFirstDigit(targetDigit);
+      this.renderTotal(this.firstDigit);
+    }
+    // 두번째 수 입력
+    if (this.validateSecondDigit(targetDigit)) {
+      this.updateSecondDigit(targetDigit);
+      this.renderTotal(this.firstDigit + this.operation + this.secondDigit);
+    }
+  }
+
+  handleOperation(e) {
+    if (e.target.textContent === "=") {
+      this.handleOperationEqual();
+      return;
+    }
+
+    if (!this.checkMoreThanTwoNumber()) {
+      return;
+    }
+
+    this.operation = e.target.textContent;
```

Comment:
this.operation 을 53번째줄 위로 옮기시면 좋을 것 같아요.
```suggestion
    this.operation = e.target.textContent;
    if (this.operation === "=") {
      this.handleOperationEqual();
      return;
    }

    if (!this.checkMoreThanTwoNumber()) {
      return;
    }

```